examples/RNOneSignal/android/app/src/main/java/com/rnonesignal/MainApplication.java

```diff
@@ -25,8 +22,7 @@ public boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage(),
-            new ReactNativeOneSignalPackage()
```

Comment:
I believe `new ReactNativeOneSignalPackage()` should still be here, otherwise the native part of the SDK won't be started
Does the example app still register the device on Android?